Source/OxyPlot/Series/VectorSeries.cs

```diff
@@ -0,0 +1,569 @@
+        /// </summary>
+        /// <remarks>
+        /// A value of 0 indicates the arrow will start at the <see cref="VectorItem.Origin"/>.
+        /// A value of 1 indicates the arrow will terminate at the <see cref="VectorItem.Origin"/>.
+        /// Values other than 0 and 1 may produce arrows that do not pass through the <see cref="VectorItem.Origin"/> on non-linear axes.
+        /// </remarks>
+        public double ArrowStartPosition { get; set; }
+
+        /// <summary>
+        /// Gets the psoitions of the label for each vector along the drawn arrow (the default value is 0).
+        /// </summary>
+        /// <remarks>
+        /// A value of 0 indicates the label will be positioned at the start of the vector arrow.
+        /// A value of 1 indicates the label will be positioned at the end of the vector arrow.
+        /// </remarks>
+        public double ArrowLabelPosition { get; set; }
+
+        /// <summary>
+        /// Gets or sets the color axis.
+        /// </summary>
+        /// <value>The color axis.</value>
+        public IColorAxis ColorAxis { get; protected set; }
+
+        /// <summary>
+        /// Gets or sets the color axis key.
+        /// </summary>
+        /// <value>The color axis key.</value>
+        public string ColorAxisKey { get; set; }
+
+        /// <summary>
+        /// Gets or sets the format string for the cell labels. The default value is <c>0.00</c>.
+        /// </summary>
+        /// <value>The format string.</value>
+        /// <remarks>The label format string is only used when <see cref="LabelFontSize" /> is greater than 0.</remarks>
+        public string LabelFormatString { get; set; }
+
+        /// <summary>
+        /// Gets or sets the font size of the labels. The default value is <c>0</c> (labels not visible).
+        /// </summary>
+        /// <value>The font size relative to the cell height.</value>
+        public double LabelFontSize { get; set; }
+
+        /// <summary>
+        /// Gets or sets a value indicating whether the tracker can interpolate points.
+        /// </summary>
+        public bool CanTrackerInterpolatePoints { get; set; }
+
+        /// <summary>
+        /// Gets or sets the delegate used to map from <see cref="ItemsSeries.ItemsSource" /> to <see cref="VectorItem" />. The default is <c>null</c>.
+        /// </summary>
+        /// <value>The mapping.</value>
+        /// <remarks>Example: series1.Mapping = item => new VectorItem(new DataPoint((MyType)item).Time1, ((MyType)item).Value1), new DataPoint((MyType)item).Time2, ((MyType)item).Value2));</remarks>
+        public Func<object, VectorItem> Mapping { get; set; }
+
+        /// <summary>
+        /// Gets the list of Vectors.
+        /// </summary>
+        /// <value>A list of <see cref="VectorItem" />. This list is used if <see cref="ItemsSeries.ItemsSource" /> is not set.</value>
+        public List<VectorItem> Items { get; } = new List<VectorItem>();
```

Comment:
I would prefer to expose an `IList<VectorItem>`. I know we expose `List`s elsewhere, so it would be somewhat inconsistent, but as far as I am concerned it is still preferable.